### PR TITLE
clean post 3.11 merge

### PR DESCRIPTION
**Pull request by mjudeikis**
OSA 3.11 post-merge fixes

/cc @kargakis

### CODE REVIEW

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 1]
```diff
@@ -179,9 +173,7 @@ objects:
         echo "Using logbridge image ${LOGBRIDGE_IMAGE}"
         export DNS_DOMAIN=osadev.cloud
         export DNS_RESOURCEGROUP=dns
-        export DEPLOY_VERSION=${DEPLOY_VERSION}
-        export IMAGE_OFFER=${OSA_IMAGE_OFFER}
-        export IMAGE_VERSION=${OSA_IMAGE_VERSION}
+        export DEPLOY_VERSION=${DEPLOY_VERSION}--config /release/ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11.yaml \
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: what's up with the --config part?

Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: This is not the change you are looking for...

**Pull request merged**